CRM/Dedupe/Finder.php

```diff
@@ -360,15 +358,7 @@ private static function getTree($entityType, $subTypes, $params) {
       foreach ($subTypes as $subType) {
         $filters['extends_entity_column_value'][] = self::validateSubTypeByEntity($entityType, $subType);
       }
-      if (!$onlySubType) {
-        $filters['extends_entity_column_value'][] = NULL;
-      }
-      if ($subName) {
-        $filters['extends_entity_column_id'] = $subName;
-      }
-    }
-    elseif (!$returnAll) {
-      $filters['extends_entity_column_value'] = NULL;
+      $filters['extends_entity_column_value'][] = NULL;
```

Comment:
this is the line from 364 - the other 2 were removed. I can remove returnAll too now from the looks but will do another pass